CODEOWNERS

```diff
@@ -15,7 +15,7 @@ azure-pipelines.yml @garyverhaegen-da
 azure-cron.yml      @garyverhaegen-da
 /release/           @garyverhaegen-da
 /release.sh         @garyverhaegen-da
-/canton_dep.bzl     @garyverhaegen-da
+/canton_dep.bzl     @garyverhaegen-da @remyhaemmerle-da
```

Comment:
@rgugliel-da @phoebe-da Do you want to be codeowners as well ? 
Like that you will automatically added as reviewer for the automatic bump of canton in the main repo ?

Reply:
like that you will be notify for each update of canton in the daml repo.

Reply:
And to clarify: in this repo, we use CODEOWNERS _only_ as a notification mechanism, this wouldn't give you any extra responsibility.

Reply:
Yes, that would be nice, thanks!

Reply:
Added. I'll give @phoebe-da some time to answer, but worst case we can make another PR later on.